.github/ISSUE_TEMPLATE.md

```diff
@@ -1,89 +1,48 @@
-
--->
-
-### Description
-
-[Description of the issue]
-
-### Version
 
 <!--
-
-What version of GitHub Desktop are you running? This is displayed under the
-`About GitHub Desktop` menu item. If you are running from source, include
-the commit by running `git rev-parse HEAD` from your local repository.
-
+First and foremost, we’d like to thank you for taking the time to contribute to our project. Before submitting your issue, please follow these steps:
+
+1. Familiarize yourself with our contributing guide:
+	* https://github.com/desktop/desktop/blob/master/CONTRIBUTING.md#contributing-to-github-desktop
+2. Check if your issue is in the known-issues and work-arounds doc:
+	* [Need link]
+3. Make sure your issue isn’t a duplicate of another issue
+4. If you have made it to this step, go ahead and fill out the template below
 -->
+## Description
+<!-- Provide a detailed description of the behavior you're seeing or the behavior you'd like to see. -->
+## Version
+* GitHub Desktop: <!-- Replace with the version of GitHub Desktop you have installed -->
+* Operating system: <!-- Replace with the version of your operating system -->
 
-**GitHub Desktop version:** [version here]
-
+## Steps to Reproduce
 <!--
+1. `step 1`
+2. `step 2`
+3. `and so on…`
 
-The operating system you are running on may also help with reproducing the
-issue:
-
- - If you are on macOS, launch `About This Mac` and write down the OS version
-   listed.
- - If you are on Windows, open `Command Prompt` and attach the output of this
-   command: `cmd /c ver`
-
+Include screen shots or gifs if available
 -->
 
-**OS version:** [version here]
-
-### Steps to Reproduce
-
-1. [First step]
-1. [Second step]
-1. [and so on]
-
-<!--
-
-If the issue involves a specific public repository, including the information
-about that repository will make it is easier to recreate the issue.
-
-If you think screenshots or a GIF recording will help demonstrate the issue
-better, feel free to add them here.
-
--->
-
-**Expected behavior:** [What you expected to happen]
-
-**Actual behavior:** [What actually happened]
-
-**Reproduces how often:** [What percentage of the time does it reproduce?]
-
+### Expected Behavior
+<!-- Replace with what you expected to happen -->
+### Actual Behavior
+<!-- Replace with what actually happens -->
+### How often does this issue occur?
+<!-- Sometimes`, `Most of the time`, or `Every time -->
+## Additional Information
+<!-- 
+Replace with any additional information, configuration, or data that might be necessary to reproduce the issue.
+
+If you are dealing with a performance issue or regression, attaching a Timeline profile of the task will help the developers understand the runtime behavior of the application on your machine.
+	* https://github.com/desktop/desktop/blob/master/docs/contributing/timeline-profile.md
+ -->
 ### Logs
+<!-- 
+Replace with your log file (You can simply drag your file here to insert it) 
```

Comment:
Will it insert if they drag it inside the comment accidentally? Even if it does insert, it probably won't render? Maybe consider rephrasing to  instruct them to drop it "below" or something?